code/modules/cargo/packs.dm

```diff
@@ -534,13 +543,32 @@
 /datum/supply_pack/security/armory/wt550ammo
 	name = "WT-550 Auto Rifle Ammo Crate"
 	desc = "Contains four 20-round magazines for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
-	cost = 3000
+	cost = 2500
 	contains = list(/obj/item/ammo_box/magazine/wt550m9,
 					/obj/item/ammo_box/magazine/wt550m9,
 					/obj/item/ammo_box/magazine/wt550m9,
 					/obj/item/ammo_box/magazine/wt550m9)
 	crate_name = "auto rifle ammo crate"
 
+/datum/supply_pack/security/armory/wt550ammo_nonlethal // Takes around 11 shots to stun crit someone
+	name = "WT-550 Auto Rifle Non-Lethal Ammo Crate"
+	desc = "Contains four 20-round magazines for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
+	cost = 1500
+	contains = list(/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber,
+					/obj/item/ammo_box/magazine/wt550m9/wtrubber)
+	crate_name = "auto rifle ammo crate"
+
+/datum/supply_pack/security/armory/wt550ammo_special
+	name = "WT-550 Auto Rifle Special Ammo Crate"
+	desc = "Contains 2 20-round Armour Piercing and Incendiary magazines for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads. Requires Armory access to open."
+	cost = 3000
```

Comment:
Pretty cheap for some pretty deadly rounds

Reply:
KK ill up the cost to 4500